packages/phoenix-client/src/phoenix/client/resources/spans/__init__.py

```diff
@@ -0,0 +1,272 @@
+import logging
+from datetime import datetime
+from io import StringIO
+from typing import TYPE_CHECKING, Optional, cast
+
+import httpx
+
+from phoenix.datetime_utils import normalize_datetime
```

Comment:
I don't think you can import this from this low down can you? Don't you have to copy this over?